src/Bridges/set_map.jl

```diff
@@ -5,35 +5,74 @@
 # in the LICENSE.md file or at https://opensource.org/licenses/MIT.
 
 """
+    struct MapNotInvertible <: Exception
+        message::String
+    end
+
+An error thrown by [`inverse_map_function`](@ref) or
+[`inverse_adjoint_map_function`](@ref) indicating that the linear map `A`
+defined in [`Variable.SetMapBridge`](@ref) and [`Constraint.SetMapBridge`](@ref)
+is not invertible.
+"""
+struct MapNotInvertible <: Exception
+    message::String
+end
+
+"""
+    map_set(bridge::MOI.Bridges.AbstractBridge, set)
     map_set(::Type{BT}, set) where {BT}
 
 Return the image of `set` through the linear map `A` defined in
-[`Variable.SetMapBridge`](@ref) and [`Constraint.SetMapBridge`](@ref). This is
-used for bridging the constraint and setting
-the [`MOI.ConstraintSet`](@ref).
+[`Variable.SetMapBridge`](@ref) and [`Constraint.SetMapBridge`](@ref).
+
+This function is used for bridging the constraint and setting the
+[`MOI.ConstraintSet`](@ref).
+
+The default implementation of [`Variable.bridge_constraint`](@ref) uses
+[`map_set`](@ref) with the bridge type so if this function is defined
+on the bridge type, [`Variable.bridge_constraint`](@ref) does not need
+to be implemented.
```

Comment:
I don't understand this comment. `Variable.bridge_constraint` does not exist.

Reply:
Do you mean
https://github.com/jump-dev/MathOptInterface.jl/blob/193aec66d562c37af5e63f842652dc44a31bce9a/src/Bridges/Constraint/set_map.jl#L37-L45

Reply:
I just removed the comment